resin/resin/event/window_events.hpp

```diff
@@ -10,11 +10,20 @@ class WindowCloseEvent : public Event<EventType::WindowCloseEvent> {
  public:
   EVENT_NAME(WindowCloseEvent)
 
-  std::string to_string() const override { return name(); }
+  std::string to_string() const override { return std::string(name()); }
```

Comment:
`name()` returns `std::string_view`, so the `to_string` could also return the `std::string_view`, no need for the additional heap allocation, the caller may use the `std::string(...)` if they want to become an owner

Reply:
I vividly remember changing this to string_view. I must've dreamt it lol

Reply:
actually not viable due to 
```c++
std::string WindowResizeEvent::to_string() const override {
  std::stringstream ss;
  ss << name() << ": " << width_ << " x " << height_;
  return ss.str();
}
```
which has to remain a string